js/battle.js

```diff
@@ -955,11 +955,19 @@ var Sprite = (function () {
 		}
 	};
 	Sprite.prototype.animSummon = function (slot, instant) {
-		this.x = slot * (this.isBackSprite ? -1 : 1) * -50;
-		this.y = slot * (this.isBackSprite ? -1 : 1) * 10;
-		this.statbarOffset = 0;
-		if (!this.isBackSprite) this.statbarOffset = 17 * slot;
-		if (this.isBackSprite) this.statbarOffset = -7 * slot;
+		if(!Tools.prefs('nopastgens') && this.battle.gen <= 4 && this.battle.gameType === 'doubles'){
```

Comment:
There needs to be a space between `if` and `(`, and another space between `)` and `{`.